src/components/quiz/questions.ts

```diff
@@ -0,0 +1,91 @@
+export interface Question {
+  id: number;
+  Question: string;
+  Answers: string[];
+  Answer: number;
+  details: string;
+  url: string;
+}
+
+export const questions: Question[] = [
+  {
+    "id": 1,
+    "Question": "How many moons does Mars have?",
+    "Answers": ["none", "1", "2"],
+    "Answer": 2,
+    "details": "Mars has 2 moons. They are named Phobos, meaning fear and Deimos, which is greek for panik.",
+    "url": "https://svs.gsfc.nasa.gov/vis/a010000/a011300/a011326/marsmoons_1024x576.jpg"
+  },
+  {
+    "id": 2,
+    "Question": "What is Mars’s nickname?",
+    "Answers": ["The red Planet", "The dusty planet", "The stinky planet"],
+    "Answer": 0,
+    "details": "Rusted iron particles in rocks give mars it's distinct red hue",
+    "url": "https://www.nasa.gov/sites/default/files/styles/full_width_feature/public/thumbnails/image/pia19393-main_sol3973_spirit_of_st_louis_l257atc.jpg"
+  },
+  {
+    "id": 3,
+    "Question": "About how long would it take to travel to Mars? ",
+    "Answers": ["Three days", "One week", "Eight months"],
+    "Answer": 2,
+    "details": "Earth and Mars follow different orbits around the Sun, so space explorers must wait until the two planets are close together to blast off.Even then, the trip would take a minimum of about eight months.",
+    "url": "https://www.nasa.gov/sites/default/files/thumbnails/image/48181678987_dd26a6ed67_k.jpg"
+  },
+  {
+    "id": 4,
+    "Question": "True or false? Scientists have found evidence of liquid water on Mars. ",
+    "Answers": ["true", "false"],
```

Comment:
These will be displayed to the reader, so should be capitalised:
```suggestion
    "Answers": ["True", "False"],
```